docs/build/cosmos-evm.md

```diff
@@ -0,0 +1,173 @@
+# Cosmos Ethermint (EVM) Support
+
+We provide a custom data source processor for [Cosmos's Ethermint EVM](https://github.com/cosmos/ethermint). This offers a simple way to filter and index both EVM and Cosmos activity on many Cosmos networks within a single SubQuery project.
+
+**You can also refer to the basic [Cronos EVM](https://github.com/subquery/tutorials-frontier-evm-starter/tree/moonriver) example project with an event and call handler.**
```

Comment:
Should this be a different link? 

I just found this on Naveens github https://github.com/DeveloperInProgress/ethermint-evm-starter